config/default.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
     // AWS_SECRET_ACCESS_KEY: process.env.AWS_FAKE_KEY,
     AWS_REGION: process.env.AWS_REGION || 'us-east-1',
     IS_LOCAL_DB: process.env.IS_LOCAL_DB ? process.env.IS_LOCAL_DB === 'true' : true,
-    URL: process.env.DYNAMODB_URL || 'http://localhost:8000',
+    DYNAMODB_URL: process.env.DYNAMODB_URL || 'http://localhost:8000',
```

Comment:
Not sure what this change is for. Did you change all the uses of this value throughout the app?

Reply:
In all places it tries to get Dynamo DB url using "DYNAMODB.DYNAMODB_URL" path and I need to change only this file.